### PR TITLE
Prepare changelog for release 0.4.2

### PR DESCRIPTION
**Pull request by ben-hamelin**
## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes/no
| Did you apply meaningful labels to the pull request? | yes/no
| Did you perform a self review first? | yes/no
| Documentation reflects changes? | yes/no
| `CHANGELOG` reflects changes? | yes/no
| Unit/Functional tests reflect changes? | yes/no
| Did you perform browser testing? | yes/no
| Risk level | Low, Medium, High
| Relevant links | JIRA issue, bug reports, security alerts, etc.

### CODE REVIEW

**Review by jordan-caldwell — CHANGES_REQUESTED** [review]
I think you've got two unreleased sections and not one with date / tag

**Comment by ben-hamelin** [comment]
🤦🏻

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,22 +10,31 @@ modified Semantic Versioning scheme. See the "Versioning scheme" section of the
...
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security
+
+## [0.4.2] - 2021-01-07
```
Comment (jordan-caldwell) [inline CHANGELOG.md:24]: Today is the 11th. :back-away-slowly:

**Review by jordan-caldwell — CHANGES_REQUESTED** [review]

**Review by jordan-caldwell — APPROVED** [review]
👍 💯  /approved

**Comment by ben-hamelin** [comment]
Thank you - sorry for the hack, I retire......

**Pull request merged**